### DISCUSSION TITLE
Index Patterns in SO2

### DISCUSSION THREAD

**Discussion by squares-ui** [Security-Onion-Solutions/securityonion#2523] [Unsupported Versions] [answered]
I notice in SO2 that index patterns have been reduced from several to just "so-*"

I can see the benefit of this, but if I am just wanting to look at zeek logs over 5 days it puts me in trouble:
I can't specify one index (as it's multiple days)
And I get lots of background noise from other engines in SO

I can see in Kibana we are able to add indexes back in (e.g. so-zeek-*), but I'm conscious this goes against your design.

Are there anyway other way to filter by source?

Many thanks

**Comment by TOoSmOotH** [comment, marked as answer]
so-* is the catch all for all things Security Onion. It doesn't go against anything if you want to limit your search to zeek data by using so-zeek-* as the index or using filters in KQL to restrict to zeek. Either way will work.

**Comment by squares-ui** [comment]
Brill thanks, I'll put the indexPatterns back in (for this usecase)

Andy

**Comment by squares-ui** [comment]
Or event.module works too
